api/grpcserver.go

```diff
@@ -273,100 +273,122 @@ func (svr *GRPCServer) ReadState(ctx context.Context, in *iotexapi.ReadStateRequ
 
 // EstimateGasForAction estimates gas for action
 func (svr *GRPCServer) EstimateGasForAction(ctx context.Context, in *iotexapi.EstimateGasForActionRequest) (*iotexapi.EstimateGasForActionResponse, error) {
-	estimateGas, err := svr.coreService.EstimateGasForAction(in.Action)
+	selp, err := (&action.Deserializer{}).ActionToSealedEnvelope(in.Action)
```

Comment:
no need to change this file, see comments above

Reply:
revert.